config/config.js

```diff
@@ -17,6 +17,7 @@ module.exports = {
   logFilePath: path.join(riseCachePath,"risecache.log"),
   diskThreshold: 500 * 1024 * 1024,  // 500 MB
   fileUpdateDuration: 1200000, // 20 minutes,
+  requestTimeout: 300000, // 5 minutes
```

Comment:
@donnapep It seems a really long time to allow for and I don't remember the discussion on deciding 20 mins, so I opted for 5. Was there a reason for 20 mins?